README.md

```diff
@@ -7,27 +7,36 @@ Common functionality shared throughout node-clinic.
 
 <!-- MarkdownTOC autolink="true" levels="1,2,3,4" -->
 
-- [Utils](#utils)
-  - [`getLoggingPaths(toolName, toolSpecificFiles=[])`](#getloggingpathstoolname-toolspecificfiles)
-- [Scripts](#scripts)
-  - [`buildJs(opts = {})`](#buildjsopts--)
+- [node-clinic-common](#node-clinic-common)
+  - [Utils](#utils)
+    - [`getLoggingPaths(toolName, toolSpecificFiles=[])`](#getloggingpathstoolname-toolspecificfiles)
+  - [Scripts](#scripts)
+    - [`buildJs(opts = {})`](#buildjsopts)
+      - [Usage](#usage)
+    - [`buildCss(opts = {})`](#buildcssopts)
+      - [Usage](#usage)
+  - [Templates](#templates)
+    - [`mainTemplate(opts = {})`](#maintemplateopts)
+      - [Usage](#usage)
+  - [Styles](#styles)
     - [Usage](#usage)
-  - [`buildCss(opts = {})`](#buildcssopts--)
-    - [Usage](#usage-1)
-- [Templates](#templates)
-  - [`mainTemplate(opts = {})`](#maintemplateopts--)
-    - [Usage](#usage-2)
-- [Styles](#styles)
-  - [Usage](#usage-3)
-  - [Standards](#standards)
-    - [Breakdown](#breakdown)
-- [Behaviours](#behaviours)
-  - [`ask()`](#ask)
-  - [Usage](#usage-4)
-- [Icons](#icons)
-- [Components](#components)
+    - [Standards](#standards)
+      - [Breakdown](#breakdown)
+  - [Behaviours](#behaviours)
+    - [`ask()`](#ask)
+    - [Usage](#usage)
+  - [Icons](#icons)
+  - [Components](#components)
+    - [button](#button)
+    - [link](#link)
+    - [dropdown](#dropdown)
+    - [checkbox](#checkbox)
+    - [contexOverlay](#contexoverlay)
+    - [Walkthrough](#walkthrough)
+    - [howDoesThisWork](#howdoesthiswork)
```

Comment:
Could we call this eerywhere to something relating to what it *is* or *does* (e.g. `walkthroughButton` or `activateWalkthrough` etc), which won't change, rather than current wording of user-facing label text which often does change?